mvlearn/compose/wrap.py

```diff
@@ -0,0 +1,157 @@
+"""Singleview function wrapping utilities."""
```

Comment:
this wrapper is very similar to `mvlearn.preprocessing.ViewTransformer`, so maybe both should be put in the same file?

Reply:
Yes, that was my thought. Here, I am providing a wrapper for estimators with the `predict` function which doesn't belong in a preprocessing module. I put it in `compose` as it's composing a multiview function from sklearn single view ones. I propose moving `ViewTransformer` to `compose`. thoughts?

Reply:
Yes I think it's the most sensible thing to do.